README.md

```diff
@@ -49,37 +49,53 @@ Lino::CommandLineBuilder.for_command('ls')
     .with_flag('-a')
     .build
     .to_s
-    
+
 # => ls -l -a
-  
+
 # commands with options
 Lino::CommandLineBuilder.for_command('gpg')
     .with_option('--recipient', 'tobyclemson@gmail.com')
     .with_option('--sign', './doc.txt')
     .build
-    .to_s 
-    
+    .to_s
+
 # => gpg --recipient tobyclemson@gmail.com --sign ./doc.txt
-  
-# commands with arguments
+
+# commands with an option repeated multiple times (nil and empty option values are ignored)
+Lino::CommandLineBuilder.for_command('example.sh')
+    .with_repeated_option('--opt', ['file1.txt', nil, '', 'file2.txt'])
+    .build
+    .to_s
+
+# => example.sh --opt file1.txt --opt file2.txt
+
+# commands with arguments 
 Lino::CommandLineBuilder.for_command('diff')
     .with_argument('./file1.txt')
     .with_argument('./file2.txt')
     .build
-    .to_s 
-    
+    .to_s
+
 # => diff ./file1.txt ./file2.txt
-  
+
+# commands with an array of arguments (nil and empty arguments are ignored)
```

Comment:
Inclined to remove the bracketed section of the comment (and the nil and '' values in the example)..

Reply:
Agreed, I think the example is self-explanatory